x-pack/test/detection_engine_api_integration/security_and_spaces/tests/import_export_rules.ts

```diff
@@ -0,0 +1,278 @@
+        .expect(200);
+
+      await createRule(supertest, log, {
+        actions: [action],
+        name: 'Query with all possible fields filled out',
+        description: 'Kitchen Sink (everything) query that has all possible fields filled out',
+        false_positives: [
+          'https://www.example.com/some-article-about-a-false-positive',
+          'some text string about why another condition could be a false positive',
+        ],
+        rule_id: 'rule-id-everything',
+        filters: [
+          {
+            query: {
+              match_phrase: {
+                'host.name': 'siem-windows',
+              },
+            },
+          },
+          {
+            exists: {
+              field: 'host.hostname',
+            },
+          },
+        ],
+        enabled: false,
+        exceptions_list: [
+          {
+            id: exceptionBody.id,
+            list_id: exceptionBody.list_id,
+            type: exceptionBody.type,
+            namespace_type: exceptionBody.namespace_type,
+          },
+        ],
+        index: ['auditbeat-*', 'filebeat-*'],
+        interval: '5m',
+        query: 'user.name: root or user.name: admin',
+        output_index: '.siem-signals-default',
+        meta: {
+          anything_you_want_ui_related_or_otherwise: {
+            as_deep_structured_as_you_need: {
+              any_data_type: {},
+            },
+          },
+        },
+        language: 'kuery',
+        risk_score: 1,
+        max_signals: 100,
+        tags: ['tag 1', 'tag 2', 'any tag you want'],
+        to: 'now',
+        from: 'now-6m',
+        severity: 'high',
+        type: 'query',
+        threat: [
+          {
+            framework: 'MITRE ATT&CK',
+            tactic: {
+              id: 'TA0040',
+              name: 'impact',
+              reference: 'https://attack.mitre.org/tactics/TA0040/',
+            },
+            technique: [
+              {
+                id: 'T1499',
+                name: 'endpoint denial of service',
+                reference: 'https://attack.mitre.org/techniques/T1499/',
+              },
+            ],
+          },
+          {
+            framework: 'Some other Framework you want',
+            tactic: {
+              id: 'some-other-id',
+              name: 'Some other name',
+              reference: 'https://example.com',
+            },
+            technique: [
+              {
+                id: 'some-other-id',
+                name: 'some other technique name',
+                reference: 'https://example.com',
+              },
+            ],
+          },
+        ],
+        references: [
+          'http://www.example.com/some-article-about-attack',
+          'Some plain text string here explaining why this is a valid thing to look out for',
+        ],
+        timeline_id: 'timeline_id',
+        timeline_title: 'timeline_title',
+        note: '# note markdown',
+        version: 1,
```

Comment:
nit: Theres a lot of example rules to override, I wouldn't use the `kitchen sink` example here. Also it's using fields we don't like anymore such as:

```
 output_index: '.siem-signals-default',
```

I would either pull a sample rule out and then splat/override/modify it, or just try this example way down to remove parts we don't want. If it starts failing due to no fault of yours it could be one of these other fields you don't really care about. If you want to test multiple things, we should split it up to try and see if we can have 1 smaller rule vs another smaller rule to test each thing independent.